Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedListItemView.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+public struct PreplannedListItemView: View {
+    /// The view model for the preplanned map.
+    @ObservedObject var preplannedMapModel: PreplannedMapModel
+    /// A Boolean value indicating whether the preplanned map area can be downloaded.
+    @State private var canDownload: Bool?
+    /// The error for the preplanned map.
+    @State var error: Error?
+    
+    public var body: some View {
+        HStack {
+            HStack {
+                let preplannedMapArea = preplannedMapModel.preplannedMapArea
+                if let thumbnail = preplannedMapModel.preplannedMapArea.portalItem.thumbnail {
+                    LoadableImageView(loadableImage: thumbnail)
+                        .frame(width: 64, height: 44)
+                }
+                VStack(alignment: .leading, spacing: 2) {
+                    Text(preplannedMapArea.portalItem.title)
+                        .font(.headline)
+                    if !preplannedMapArea.portalItem.description.isEmpty {
+                        Text(preplannedMapArea.portalItem.description)
+                            .font(.caption)
+                            .foregroundColor(.secondary)
+                            .lineLimit(2)
+                    }
```

Comment:
I feel we can use a separate text for area with no description as description is not a required non-empty field. Something like

```swift
if !preplannedMapArea.portalItem.description.isEmpty {
    Text(preplannedMapArea.portalItem.description)
        .font(.caption)
        .foregroundStyle(Color(uiColor: .secondaryLabel))
        .lineLimit(2)
} else {
    Text("The area has no description.")
        .font(.caption)
        .foregroundStyle(Color(uiColor: .tertiaryLabel))
}
```